content/docs/user-guide/components/reference/gradients/gradient-baker.md

```diff
@@ -0,0 +1,56 @@
+---
+linktitle: Gradient Baker
+title: Gradient Baker Component
+description: Use the Gradient Baker component to bake a complex node graph of gradients down to a single static image in Open 3D Engine (O3DE).
+---
+
+Use the **Gradient Baker** component to bake a complex node graph of gradients down to a single static image. You can then use a single [Image Gradient](/docs/user-guide/components/reference/gradients/image-gradient/) at runtime, instead of having the entire node graph computed every time, to optimize performance.
+
+For example, take this complex gradient chain with a generator and multiple gradient modifiers.
+
+![Complex chain without gradient baker](/images/user-guide/components/reference/gradients/gradient-baker-chain-without-baker.png)
+
+You can create a **Gradient Baker** and connect its input gradient to the output of the gradient chain you'd like to bake. The **Gradient Baker** also needs an input bounds for where to sample the data. With both of those connected, the graph would look like this.
+
+![Complex chain with gradient baker](/images/user-guide/components/reference/gradients/gradient-baker-chain-with-baker.png)
```

Comment:
Regarding my comment about using only the second image for both paragraphs, here's what I suggest.

```suggestion
For example, suppose you have a complex gradient chain with a generator and multiple gradient modifiers.
You can create a **Gradient Baker** and connect its input gradient to the output of the gradient chain you'd like to bake. The **Gradient Baker** also needs an input bounds for where to sample the data. With both of those connected, the graph would look like this.

![Complex chain with gradient baker](/images/user-guide/components/reference/gradients/gradient-baker-chain-with-baker.png)
```

Reply:
Awesome thanks, will do this!